audio/speech/getting-started/get_started_with_chirp_2_sdk.ipynb

```diff
@@ -0,0 +1,1331 @@
+      "source": [
+        "### Perform batch speech recognition"
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "LJxhFSg848MO"
+      },
+      "source": [
+        "#### Define batch recognition configuration\n",
+        "\n",
+        "You start defining the batch recognition configuration."
+      ]
+    },
+    {
+      "cell_type": "code",
+      "execution_count": null,
+      "metadata": {
+        "id": "0CEQUL5_5BT-"
+      },
+      "outputs": [],
+      "source": [
+        "batch_recognition_config = cloud_speech.RecognitionConfig(\n",
+        "    language_codes=[\"en-US\"],\n",
+        "    model=\"chirp_2\",\n",
+        "    features=cloud_speech.RecognitionFeatures(\n",
+        "        enable_automatic_punctuation=True,\n",
+        "    ),\n",
+        "    auto_decoding_config=cloud_speech.AutoDetectDecodingConfig(),\n",
+        ")"
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "SKf3pMBl5E4f"
+      },
+      "source": [
+        "#### Set the audio file you want to transcribe\n",
+        "\n",
+        "For the batch transcription, you need the audio be staged in a Cloud Storage bucket. Then you set the associated metadata to pass in the batch recognition request."
+      ]
+    },
+    {
+      "cell_type": "code",
+      "execution_count": null,
+      "metadata": {
+        "id": "o1VCvEEI5MkG"
+      },
+      "outputs": [],
+      "source": [
+        "audio_metadata = cloud_speech.BatchRecognizeFileMetadata(\n",
+        "    uri=INPUT_LONG_AUDIO_SAMPLE_FILE_URI\n",
+        ")"
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "5HOKZLp25yFB"
+      },
+      "source": [
+        "#### Define batch recognition request\n",
+        "\n",
+        "Next, you define the batch recognition request. Notice how you define a recognition output configuration which allows you to determine how would you retrieve the resulting transcription outcome."
+      ]
+    },
+    {
+      "cell_type": "code",
+      "execution_count": null,
+      "metadata": {
+        "id": "SItkaX7tyZ14"
+      },
+      "outputs": [],
+      "source": [
+        "batch_recognition_request = cloud_speech.BatchRecognizeRequest(\n",
+        "    recognizer=f\"projects/{PROJECT_ID}/locations/{LOCATION}/recognizers/_\",\n",
+        "    config=batch_recognition_config,\n",
+        "    files=[audio_metadata],\n",
+        "    recognition_output_config=cloud_speech.RecognitionOutputConfig(\n",
+        "        inline_response_config=cloud_speech.InlineOutputConfig(),\n",
+        "    ),\n",
+        ")"
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "YQY1eqaY7H0n"
+      },
+      "source": [
+        "#### Run the batch recognition request\n",
+        "\n",
+        "Finally you submit the batch recognition request which is a long term operation as you see below."
```

Comment:
```suggestion
        "Finally you submit the batch recognition request which is a long-running operation as you see below."
```

Maybe add a link to the docs about long-running operations? https://google.aip.dev/151